Update Private_class_fields (#30033)
mdn/content: files/en-us/web/javascript/reference/classes/private_properties/index.md

---
title: Private properties
slug: Web/JavaScript/Reference/Classes/Private_properties
page-type: javascript-language-feature
browser-compat: javascript.classes.private_class_fields
---

{{jsSidebar("Classes")}}

> Private properties are counterparts of the regular class properties that are [public](/en-US/docs/Web/JavaScript/Reference/Classes/Public_class_fields) by default. Private properties cannot be accessed from outside of the class. They get created by using a hash `#` prefix. The privacy encapsulation of these class features is enforced by JavaScript itself.

Reviewer: ```suggestion
**Private properties** are counterparts of the regular class properties that are public, including [class fields](/en-US/docs/Web/JavaScript/Reference/Classes/Public_class_fields), class methods, etc. They get created by using a hash `#` prefix and cannot be legally referenced outside of the class. The privacy encapsulation of these class features is enforced by JavaScript itself.
```

> **Private properties** are counterparts of the regular class properties that are public, including [class fields](/en-US/docs/Web/JavaScript/Reference/Classes/Public_class_fields), class methods, etc. They get created by using a hash `#` prefix and cannot be legally referenced outside of the class. The privacy encapsulation of these class features is enforced by JavaScript itself.

Reviewer: ```suggestion
**Private properties** are counterparts of the regular class properties which are public, including [class fields](/en-US/docs/Web/JavaScript/Reference/Classes/Public_class_fields), class methods, etc. Private properties get created by using a hash `#` prefix and cannot be legally referenced outside of the class. The privacy encapsulation of these class features is enforced by JavaScript itself.
```
